packages/share_plus/share_plus_platform_interface/lib/platform_interface/share_plus_platform.dart

```diff
@@ -61,12 +61,14 @@ class SharePlatform extends PlatformInterface {
     String? subject,
     String? text,
     Rect? sharePositionOrigin,
+    List<String>? fileNameOverrides,
```

Comment:
Because we are changing the platform interface, this requires a major version, otherwise this will cause issues with clients where the platform interface and the main package are not in sync.